plugins/woocommerce/src/Internal/ComingSoon/ComingSoonRequestHandler.php

```diff
@@ -66,7 +77,9 @@ function () {
 			}
 		);
 
-		include $coming_soon_template;
+		if ( ! empty( $coming_soon_template ) && file_exists( $coming_soon_template ) ) {
+			include $coming_soon_template;
+		}
```

Comment:
I think it's better to have the default behaviour - which is to not block templates from rendering when coming soon template is not available.

This means it'll render the original template, like coming soon feature is disabled.

What do you think?

```suggestion
		} else {
			return $template;
		}
```

Reply:
I'm taking this back - admin may not intend for the contents to show when the site is in coming soon mode